### PR TITLE
Fixes #47: do not install any arbitrary components by default:

### LINKED ISSUE #47: Use more reasonable defaults
At the moment, the default settings in [`main.yml`](https://github.com/geerlingguy/ansible-role-homebrew/blob/master/defaults/main.yml) install `ssh-copy-id` and `pv` packages, `homebrew/core` and `caskroom/cask` taps and the `firefox` cask.

These defaults are arbitrary at best; `main.yml` seems to me like 'active' sample code with mostly documentary purposes. The user is required to take extra action to undo those most likely unwanted settings - and for that, they need to find the file and get the syntax right.

Instead, I propose to not install any arbitrary components by default, but document how this is achieved and link to official documentation.

Pull request in the making.

### PR DESCRIPTION
**Pull request by z00m1n**
instead, document how this is achieved, link to official documentation

NOTE: I would like to have Ansible print explicitly that it skips over
package / tap / cask setup tasks if the corresponding list is empty
instead of just printing the task and then moving on; this seems more in
line with the other tasks. However, a when statement is not evaluated if
a list iterated over using with_items is empty;
see also http://stackoverflow.com/a/35471907
possibly related: https://github.com/ansible/ansible/issues/1408

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -5,18 +5,29 @@ homebrew_prefix: /usr/local
...
-  - pv
+# List of packages to be installed, e.g.
+#   homebrew_installed_packages:
+#     - ssh-copy-id
+#     - pv
+homebrew_installed_packages: []
 
 homebrew_upgrade_all_packages: no
 
-homebrew_taps:
```
Comment (elliotweiser) [inline defaults/main.yml]: I disagree with this part. IMO, those taps are sane defaults.

Comment (z00m1n) [inline defaults/main.yml]: They're redundant, as explained in the comment.

Comment (elliotweiser) [inline defaults/main.yml]: Ah-- I missed the comment. That's fine then. While I do like having `homebrew_taps` explicitly matching what gets tapped, it is probably fine to leave it empty as long as it is well documented.

Comment (elliotweiser) [inline defaults/main.yml]: Let me test this real quick and then I'll change my review

**Comment by elliotweiser** [comment]
Aw I thought this was a good PR...

**Comment by geerlingguy** [comment]
I'm not opposed to this, just haven't had the time to review and incorporate :)

I'm guessing it was closed because the branch was deleted on z00m1n's remote. Would be glad to see it again as a new PR :)

**Pull request closed without merging**